File: src/main/java/cookiedragon/luchadora/event/entity/UpdateWalkingPlayerEvent.java
package cookiedragon.luchadora.event.entity;

import cookiedragon.luchadora.event.CancellableEvent;

/**
 * @author cookiedragon234 01/Jan/2020
 */
public class UpdateWalkingPlayerEvent extends CancellableEvent
{
	public static class Pre extends UpdateWalkingPlayerEvent{}
	
	public static class Post extends UpdateWalkingPlayerEvent{}
}

File: src/main/kotlin/cookiedragon/luchadora/event/entity/UpdateWalkingPlayerEvent.kt
package cookiedragon.luchadora.event.entity

import cookiedragon.luchadora.event.CancellableEvent

/**
 * @author cookiedragon234 17/Feb/2020
 */
open class UpdateWalkingPlayerEvent : CancellableEvent() {
	class Pre : UpdateWalkingPlayerEvent()
	class Post : UpdateWalkingPlayerEvent()
}
